app/views/contracts/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1> Contracts Index </h1>
+<table class="table">
+  <thead class="thead-inverse">
+      <tr>
+        <th> ID:  </th>
```

Comment:
We don't need any of these spaces surrounding the interior content of the tags. just `<th>word</th>`